tools/twix/src/zoom_and_pan.rs

```diff
@@ -0,0 +1,43 @@
+use eframe::egui::{Response, Ui};
+use linear_algebra::IntoTransform;
+use nalgebra::{vector, Similarity2, Translation2};
+
+use crate::twix_painter::TwixPainter;
+
+#[derive(Default)]
+pub struct ZoomAndPanManager {
+    transformation: Similarity2<f32>,
+}
+
+impl ZoomAndPanManager {
```

Comment:
hmm what do you think of `ZoomAndPanTransform` instead?

Reply:
sounds good, done